torch/csrc/distributed/rpc/rref_context.h

```diff
@@ -285,6 +285,8 @@ class TORCH_API RRefContext {
   std::unordered_map<ForkId, c10::intrusive_ptr<RRef>, ForkId::Hash>
       pendingChildren_;
 
+  std::atomic<int64_t> numPendingFutures_{0};
```

Comment:
Let's add some comments to explain this metrics

Reply:
Done.